.github/workflows/tests.yml

```diff
@@ -117,7 +117,7 @@ jobs:
       - name: Check code styles
         if: runner.os != 'windows'
         run: |
-          brew install shellcheck shfmt
+          brew install shellcheck shfmt diffutils
```

Comment:
What's `diffutils` used for?

Reply:
On macOS, the BSD version `diff` command does not support option `--color=auto`. Is uncolored raw text is enough for shell script styling with `shfmt`? I could create a PR on Homebrew/brew to remove the option `--color=auto`, or keep that and install `diffutils` as a dependency for command `brew style`.

Reply:
Cool, that's good to add it for this!